src/constraint/dart/InverseKinematicsSampleable.cpp

```diff
@@ -220,7 +220,8 @@ bool IkSampleGenerator::sample(statespace::StateSpace::State* _state)
 
     mInverseKinematics->getTarget()->setTransform(poseState.getIsometry());
 
-    // Run the IK solver. If an exact solution is computed, apply it to the skeleton.
+// Run the IK solver. If an exact solution is computed, apply it to the
```

Comment:
Can we move this comment inside the `#if`? Would that help the formatter put it at the right indentation level?